src/easylink/pipeline_graph.py

```diff
@@ -4,14 +4,13 @@
 from typing import Dict, List, Tuple, Union
 
 import networkx as nx
-from networkx import MultiDiGraph
 
 from easylink.configuration import Config
-from easylink.graph_components import InputSlot
+from easylink.graph_components import ImplementationGraph, InputSlot
 from easylink.implementation import Implementation
 
 
-class PipelineGraph(MultiDiGraph):
+class PipelineGraph(ImplementationGraph):
```

Comment:
I think it would be helpful to address the answer to this question in a docstring somewhere: what is the difference between a PipelineGraph and an ImplementationGraph?